packages/fxa-content-server/app/scripts/lib/cropper.js

```diff
@@ -33,30 +30,45 @@ function Cropper(options) {
   this.left = 0;
   this.yCenter = 0;
   this.xCenter = 0;
-  this.verticalGutter = _.result(options, 'verticalGutter', DEFAULT_GUTTER);
-  this.horizontalGutter = _.result(options, 'horizontalGutter', DEFAULT_GUTTER);
   this.onRotate = options.onRotate || _.noop;
   this.onTranslate = options.onTranslate || _.noop;
   this.onZoomIn = options.onZoomIn || _.noop;
   this.onZoomOut = options.onZoomOut || _.noop;
   this.onZoomRangeChange = options.onZoomRangeChange || _.noop;
 
+  this.resizeAdjustment = _.throttle(() => {
+    this.updateMeasurements();
+
+    const pos = this.updatePosition();
+    this.img.css(pos);
+  }, 300);
+
   if (!options.container) {
     throw new Error('A container element is required');
   }
+
   this._setupElements(options.container);
 
+  window.addEventListener('resize', this.resizeAdjustment);
+
   if (options.src) {
-    var img = options;
+    let img = options;
 
     // we want the extra resolution to be able to zoom in later (up to 200%)
-    var maxLength = this.exportLength * 2;
+    const maxLength = this.exportLength * 2;
     if (img.width > maxLength || img.height > maxLength) {
-      var scale = maxLength / Math.max(img.width, img.height);
+      const scale = maxLength / Math.max(img.width, img.height);
       img = this.resize(img, scale);
     }
     this.setImageSrc(img.src, img.width, img.height);
   }
+
+  setTimeout(() => {
```

Comment:
I tried a few things here too without much luck. Can we add a comment above for justification?